conduct/conduct_enforcement.md

```diff
@@ -0,0 +1,117 @@
+* Who, if anyone, was responsible for the violation
+* Whether this is an ongoing situation, and there is a threat to anyone's
+  physical safety
+
+This information will be collected in writing, and whenever possible the group's
+deliberations will be recorded and retained (i.e. chat transcripts, email
+discussions, recorded voice conversations, etc).
+
+It is important to retain an archive of all activities of this committee to
+ensure consistency in behavior and provide institutional memory for the
+project.
+
+The Code of Conduct Committee should aim to have a resolution agreed upon within
+one week. In the event that a resolution can't be determined in that time, the
+committee will respond to the reporter(s) with an update and projected timeline
+for resolution.
+
+
+## Acting Unilaterally
+
+If the act is ongoing, or involves a threat to anyone's safety (e.g. threats of
+violence), any committee member may act immediately (before reaching consensus)
+to address the situation. In ongoing situations, any member may decide to employ
+any of the tools available to the committee, including bans and blocks.
+
+If the incident involves physical danger, any member of the committee may -- and
+should -- act unilaterally to protect the safety of those involved. This can
+include contacting law enforcement (or other local personnel) and speaking on
+behalf of the Jupyter Steering Council.
+
+In situations where an individual committee member acts unilaterally, they must
+report their actions to the committee for review within 24 hours.
+
+
+## Resolutions
+
+The committee must agree on a resolution by consensus. If the group cannot reach
+consensus and deadlocks for over a week, the group will turn the matter over to
+the Steering Council for resolution.
+
+Possible responses may include:
+
+* Taking no further action (if we determine no violation occurred).
+* A private reprimand from the committee to the individual(s) involved. In this
+  case, the group chair will deliver that reprimand to the individual(s) over
+  email, cc'ing the group.
+* A public reprimand. In this case, the committee chair will deliver that
+  reprimand in the same venue that the violation occurred (i.e. in the gitter
+  chat room for a gitter chat room violation; email for an email violation,
+  etc.). The group may choose to publish this message elsewhere for
+  documentation purposes.
+* A "mutually agreed upon hiatus" where the committee asks the individual to
+  temporarily refrain from community participation. If the individual chooses
+  not to take a temporary break voluntarily, the committee may issue a
+  "mandatory cooling off period".
+* A permanent or temporary ban from some or all Jupyter spaces (mailing lists,
+  gitter.im, etc.). The group will maintain records of all such bans so that
+  they may be reviewed in the future or otherwise maintained.
+* A request for a public or private apology. The chair will deliver this
```

Comment:
maybe move this up a few spaces so that they are more in order of degree of escalation?  I think this should go above the hiatus point.